.github/workflows/publish.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+          scope: '@dvcorg'
```

Comment:
https://github.com/iterative/link-check/blob/f132ce4a51d26d1166dbc49d0f1ee2d655ff7443/package.json#L1-L4

It doesn't look like it's scoped to `@dvcorg`.

Reply:
You're right, I totally forgot. We'll want to move it eventually, but I'll have to fix it for now.